app/controllers/inboxs_controller.rb

```diff
@@ -0,0 +1,7 @@
+class InboxsController < ApplicationController
+
+  def index
+    cm = CouncilMembership.find_by(user_id: current_user.id)
+    @questions = Question.where(council_id: cm.council_id)
```

Comment:
You may mean CouncilMemebership.where here rather than find_by. Find_by will just find the first matching record. So here I would only ever see questions sent to the first council I joined.